test/uk/gov/hmrc/cipphonenumberverification/connectors/ValidateConnectorSpec.scala

```diff
@@ -48,7 +52,13 @@ class ValidateConnectorSpec extends AnyWordSpec
       )
 
       val result = validateConnector.callService(phoneNumber)
-      status(result) shouldBe OK
+
+      def parseResponse(res: HttpResponse) = res match {
+        case r if is2xx(r.status) => Future.successful(Ok(r.body))
+        case r if is4xx(r.status) => Future.successful(BadRequest(r.body))
+      }
+
+      status(result flatMap parseResponse)
```

Comment:
what is this for?

Reply:
same question, why do we need this in test code?

Reply:
i believe this can be removed